### ISSUE TITLE
Formatting not working, shows message to see issues.

### ISSUE THREAD

**Issue by deepak-atariya-teri** [shufo/vscode-blade-formatter#14]
## Steps to Reproduce

1.
2.
3.
4.

**Comment by shufo** [comment]
@deepak-atariya-teri 
Could you provide your environment and reproduce procedure? Thanks.

**Comment by prettifystudio** [comment]
Hello, I think it doesn't work with Blade Components. I tried using it on a blade file that uses Blade Components, it doesn't work. However, using it on a "plain" blade file does the job.

**Comment by shufo** [comment]
@prettifystudio 
Could you provide example template source?
Currently custom component defined in Laravel 6.x are not supported.

Components in 7.x, 8.x like below are supported
 
```php
@section('foo')
    <x-alert />
    <x-user-profile />
    <x-inputs.button />
@endsection
```

**Comment by prettifystudio** [comment]
Laravel 8.x 
```
@push('scripts')
    <script>
      console.log('foo');
    </script>
@endpush
<x-app-layout title="Add new client">
    <section class="section">
    </section>
</x-app-layout>
```
Just a notice, the files that give me the error are the files where I'm using displaying data " {{ }} "
 Other files are working okay. I guess it's because of  " {{ }} "

**Comment by shufo** [comment]
hm...curly braces without content like `{{ }}` occurs error but if there are something in it `{{ $a }}` worked as expected. 
Thanks! I'll try figure out what causes this error.

**Comment by shufo** [comment]
Fixed in v0.6.4
